src/components/MenuSelect.js

```diff
@@ -27,27 +26,39 @@ class MenuSelect extends Component {
     };
 
     return (
-      <select
-        className={cssClasses.select}
-        value={selectedValue}
-        onChange={this.handleSelectChange}
-      >
-        <option value="" className={cssClasses.option}>
-          <Template templateKey="seeAllOption" {...templateProps} />
-        </option>
+      <div className={cssClasses.root}>
```

Comment:
Are we using [`classnames`](https://github.com/JedWatson/classnames) everywhere?